lib/wcc/contentful/store.rb

```diff
@@ -44,20 +44,23 @@ def validate!
       end
 
       def build_eager_sync(config, store = nil, *_options)
+        store = nil if [{ preview: false }, { preview: true }].include?(store)
         puts "store: #{store}"
         store = SYNC_STORES[store].call(config) if store.is_a?(Symbol)
         store || MemoryStore.new
       end
 
       def build_lazy_sync(config, *options)
+        options.delete({ preview: false })
+        options.delete({ preview: true })
```

Comment:
If the `preview` indicator is invalid for `lazy_sync`, I think it's better to raise an argument error than to silently delete the option.

Reply:
Yeah, I don't know enough about the lazy_sync to understand what you're asking for here. I agree that silent errors are bad, but you'll have to give me more context. Are you saying that this causes an error somehow?

Reply:
I was wondering why you delete the preview argument when we're building a lazy sync.  Let's circle up when you get back and put our heads together to decide whether it is or isn't valid to expec the lazy_sync strategy to use preview.